2.0/gradle/vividus-test.gradle

```diff
@@ -129,9 +129,12 @@ class StoriesExec extends VividusJavaExec {
     String fileToSaveExitCode
 }
 
+tasks.withType(VividusJavaExec).configureEach {
+    jvmArgs += defaultJvmArgs
+}
```

Comment:
why is this change needed?

Reply:
because of this flag needed not only for `StoriesExec` tasks.
For example, I caught  exception on `testVividusInitialization` and `validateKnownIssues` tasks